src/aws-fsx/ontap-file-system.ts

```diff
@@ -0,0 +1,606 @@
+import { Aws, Duration, SecretValue, Token, aws_ec2, aws_fsx } from 'aws-cdk-lib';
+import { Construct } from 'constructs';
+import { DailyAutomaticBackupStartTime } from './daily-automatic-backup-start-time';
+import { MaintenanceTime } from './maintenance-time';
+
+/**
+ * The different kinds of file system deployments used by NetApp ONTAP.
+ */
+export enum OntapDeploymentType {
+  /**
+   * A high availability file system configured for Multi-AZ redundancy to tolerate temporary Availability Zone (AZ) unavailability.
+   * This is a first-generation FSx for ONTAP file system.
+   */
+  MULTI_AZ_1 = 'MULTI_AZ_1',
+
+  /**
+   * A high availability file system configured for Multi-AZ redundancy to tolerate temporary AZ unavailability.
+   * This is a second-generation FSx for ONTAP file system.
+   */
+  MULTI_AZ_2 = 'MULTI_AZ_2',
+
+  /**
+   * A file system configured for Single-AZ redundancy.
+   * This is a first-generation FSx for ONTAP file system.
+   */
+  SINGLE_AZ_1 = 'SINGLE_AZ_1',
+
+  /**
+   * A file system configured with multiple high-availability (HA) pairs for Single-AZ redundancy.
+   * This is a second-generation FSx for ONTAP file system.
+   */
+  SINGLE_AZ_2 = 'SINGLE_AZ_2',
+}
+
+/**
+ * The throughput capacity of an Amazon FSx for NetApp ONTAP file system.
+ */
+export class ThroughputCapacityPerHaPair {
+  /**
+   * The throughput capacity of 128 MBps per HA pair.
+   */
+  public static readonly MB_PER_SEC_128 = ThroughputCapacityPerHaPair.MbPerSec(128);
```

Comment:
Could you modify it to `mbPerSec`? (because the build failed!)

```
error JSII8002: Method and property (unless they are static readonly) names must use camelCase. Rename "@open-constructs/aws-cdk.aws_fsx.ThroughputCapacityPerHaPair.MbPerSec" to "mbPerSec"
👾 Task "build » compile" failed when executing "jsii --silence-warnings=reserved-word" (cwd: /home/runner/work/aws-cdk-library/aws-cdk-library)
```